src/index.ts

```diff
@@ -173,6 +173,9 @@ export class Client {
     this.requestInstance = axios.create({
       baseURL: config.host,
       timeout: config.timeout,
+      headers: {
+        'x-atlassian-force-account-id': config.strictGDPR,
```

Comment:
@MrRefactoring this causes 'TypeError [ERR_HTTP_INVALID_HEADER_VALUE]: Invalid value "undefined" for header "x-atlassian-force-account-id"' if you don't set strictGDPR within the config. 
should be set to either true or false.
```js
headers: {
    'x-atlassian-force-account-id': config.strictGDPR || false,
}
```